src/components/Wrapper.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Wrapper.css'
+
+function Wrapper({children}) {
+  return (
+    <div className='wrapper'>
+      {children}
+    </div>
+  );
+}
+
+export default Wrapper;
```

Comment:
add empty end line

Reply:
at the end I should add empty line? for every file?